packages/playground-ui/src/ds/components/Drawer/drawer.css

```
@@ -0,0 +1,134 @@
+/* Release after a swipe: scale the settle duration by the gesture velocity. */
+.drawer-popup[data-ending-style] {
+  transition-duration: calc(var(--drawer-swipe-strength, 1) * 400ms);
+}
+
+/* ---- Bottom sheet (swipe down) ---------------------------------------------- */
+
+.drawer-popup[data-swipe-direction='down'] {
+  transform-origin: 50% calc(100% - var(--bleed));
+  transform: translateY(
+      calc(
+        var(--drawer-snap-point-offset, 0px) + var(--drawer-swipe-movement-y, 0px) - var(--stack-peek-offset) -
```

Stylelint (17.11.1)

[error] Unexpected newline after "-" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)